### ISSUE TITLE
Build documentation

### ISSUE THREAD

**Issue by BruceSherwood** [kovidgoyal/rapydscript-ng#55]
On my Mac I find that to build from Git I need to say sudo npm install, not npm install.

Also, to build the embedded compiler I have to specify a destination folder. That is, bin/web-repl-export fails. I have to say something like bin/web-repl-export embedded.

**Comment by kovidgoyal** [comment]
That seems odd. Do you have an ancient version of npm on your Mac? Quoting from the npm documentation:
```
npm install (in package directory, no arguments):                                                                                                                                                                                                      
           Install the dependencies in the local node_modules folder.
           In global mode (ie, with -g or --global appended to the command),
           it installs the current package context (ie, the current working
           directory) as a global package.
```

As for web-repl-export when you run it without arguments it tells you what to do, doesn't it?

**Comment by kovidgoyal** [comment]
To install rapydscript, you have to run 

sudo npm link .

not 

npm install

npm install installs  rapydscripts dependencies in the local folder.

Don't ask me why npm works like that.

**Comment by BruceSherwood** [comment]
As per the documentation, I ran this:

```
git clone git://github.com/kovidgoyal/rapydscript-ng.git
cd rapydscript-ng
sudo npm link .
npm install  # This will automatically install the dependencies for RapydScript
```

What I think I saw was that I had to say "sudo npm install" for the last line. But yes, it's possible I have an old version of npm; I'll check. It is true that trying to run bin/web-repl-export  gave me a message, but at first I didn't understand the problem, since I read the documentation as being explicit about the form to use.
